Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -559,9 +559,16 @@ class LeanplumStatus: HiddenSetting {
     }
 }
 
-class ClearOnboardingDefaults: HiddenSetting {
+class ClearOnboardingConstantValues: HiddenSetting {
     override var title: NSAttributedString? {
-        return NSAttributedString(string: NSLocalizedString("Debug: Clear onboarding defaults", comment: "Debug option"), attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
+        // Onboarding constant values are saved to not show the onboarding screen after user
+        // has already seen it and is dismissed. We however clear it here in this hidden setting
+        // so that when user reopens the app or when browser view controller is re-initialized
+        // we see the onboarding screen again.
+        //
+        // If we are running an A/B test this will also fetch the A/B test variables from
+        // leanplum.
+        return NSAttributedString(string: NSLocalizedString("Debug: Clear onboarding variables", comment: "Debug option"), attributes: [NSAttributedString.Key.foregroundColor: UIColor.theme.tableView.rowText])
```

Comment:
@garvankeeley Added comment to explain clearing onboarding variables and another explanation that we'll fetch A/B test variables from leanplum.